### ISSUE TITLE
How to apply new setting ?

### ISSUE THREAD

**Issue by blackyzero** [vitabaks/postgresql_cluster#167]
Hello,

After deployed successfully, how could i update postgresql config for all nodes without doing initialization?
Ex: i need to add/remove the remote host in section "postgresql_pg_hba", but i got following error when running deployment again.

```
TASK [patroni : Prepare PostgreSQL | check that data directory "/var/lib/postgresql/14/main" is not initialized] **************************************
ok: [192.168.169.151]
ok: [192.168.169.152]
ok: [192.168.169.153]

TASK [patroni : Prepare PostgreSQL | data directory check result] *************************************************************************************
fatal: [192.168.169.151]: FAILED! => {"changed": false, "msg": "Whoops! data directory /var/lib/postgresql/14/main is already initialized"}
fatal: [192.168.169.152]: FAILED! => {"changed": false, "msg": "Whoops! data directory /var/lib/postgresql/14/main is already initialized"}
fatal: [192.168.169.153]: FAILED! => {"changed": false, "msg": "Whoops! data directory /var/lib/postgresql/14/main is already initialized"}

NO MORE HOSTS LEFT ************************************************************************************************************************************

PLAY RECAP ********************************************************************************************************************************************
192.168.169.151            : ok=120  changed=3    unreachable=0    failed=1    skipped=226  rescued=0    ignored=0
192.168.169.152            : ok=118  changed=3    unreachable=0    failed=1    skipped=220  rescued=0    ignored=0
192.168.169.153            : ok=118  changed=3    unreachable=0    failed=1    skipped=220  rescued=0    ignored=0
localhost                  : ok=0    changed=0    unreachable=0    failed=0    skipped=1    rescued=0    ignored=0

```

Thank you.

**Comment by vitabaks** [comment]
Use tags - https://github.com/vitabaks/postgresql_cluster/blob/master/tags.md 

`ansible-playbook deploy_pgcluster.yml --tags pg_hba`

**Comment by blackyzero** [comment]
Great to know about tags.
I run this command, but getting following error

```
root@ansible:/opt/services/ansible/postgresql_cluster# ansible-playbook deploy_pgcluster.yml --tags pg_hba

PLAY [Deploy PostgreSQL HA Cluster (based on "Patroni" and "etcd")] ***********************************************************************************

PLAY [etcd_cluster] ***********************************************************************************************************************************

TASK [Include OS-specific variables] ******************************************************************************************************************
fatal: [192.168.169.151]: FAILED! => {"msg": "The conditional check 'not ansible_os_family == 'Rocky' and not ansible_os_family == 'AlmaLinux'' failed. The error was: error while evaluating conditional (not ansible_os_family == 'Rocky' and not ansible_os_family == 'AlmaLinux'): 'ansible_os_family' is undefined\n\nThe error appears to be in '/opt/services/ansible/postgresql_cluster/etcd_cluster.yml': line 14, column 7, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n  pre_tasks:\n    - name: Include OS-specific variables\n      ^ here\n"}
fatal: [192.168.169.152]: FAILED! => {"msg": "The conditional check 'not ansible_os_family == 'Rocky' and not ansible_os_family == 'AlmaLinux'' failed. The error was: error while evaluating conditional (not ansible_os_family == 'Rocky' and not ansible_os_family == 'AlmaLinux'): 'ansible_os_family' is undefined\n\nThe error appears to be in '/opt/services/ansible/postgresql_cluster/etcd_cluster.yml': line 14, column 7, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n  pre_tasks:\n    - name: Include OS-specific variables\n      ^ here\n"}
fatal: [192.168.169.153]: FAILED! => {"msg": "The conditional check 'not ansible_os_family == 'Rocky' and not ansible_os_family == 'AlmaLinux'' failed. The error was: error while evaluating conditional (not ansible_os_family == 'Rocky' and not ansible_os_family == 'AlmaLinux'): 'ansible_os_family' is undefined\n\nThe error appears to be in '/opt/services/ansible/postgresql_cluster/etcd_cluster.yml': line 14, column 7, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n  pre_tasks:\n    - name: Include OS-specific variables\n      ^ here\n"}

NO MORE HOSTS LEFT ************************************************************************************************************************************

PLAY RECAP ********************************************************************************************************************************************
192.168.169.151            : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
192.168.169.152            : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
192.168.169.153            : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

root@ansible:/opt/services/ansible/postgresql_cluster#
```

**Comment by vitabaks** [comment]
fixed - 6daa87575427ff35f965c91d0d777cfa21a44b06, please download the playbook again

**Comment by blackyzero** [comment]
Great. it works now.
Thank you.
